newscoop/template_engine/metaclasses/MetaLanguage.php

```diff
@@ -84,4 +85,4 @@ public static function createFromCode($code)
             $languageId = Language::GetLanguageIdByCode($code);
             return new self($languageId);
         }
-    }
\ No newline at end of file
+    }
```

Comment:
if i'm not mistaken, the diff shows i added a new line at the end of the file :)